### PR TITLE
CExplosion and some audio

### PR DESCRIPTION
**Pull request by theR4K**

### CODE REVIEW

File: src/audio/AudioLogic.cpp [hunk 1]
```diff
@@ -5933,15 +5932,15 @@ cAudioManager::ProcessGarages()
 void
 cAudioManager::ProcessFireHydrant()
 {
+	static const int SOUND_INTENSITY = 35;
```
[the remark this answers is not in the document]
Comment (theR4K) [reply src/audio/AudioLogic.cpp:5935]: I just rename it, lol

File: src/weapons/Explosion.cpp [hunk 2]
```diff
@@ -129,6 +138,7 @@ CExplosion::AddExplosion(CEntity *explodingEntity, CEntity *culprit, eExplosionT
 	explosion.m_nIteration = 1;
 	explosion.m_nActiveCounter = 1;
 	explosion.m_bIsBoat = false;
+	explosion.m_bIsMakeSound = true;
```
[the remark this answers is not in the document]
Comment (theR4K) [reply src/weapons/Explosion.cpp:141]: I'm 100% sure it was here

File: src/weapons/Explosion.cpp [hunk 3]
```diff
@@ -85,21 +86,29 @@ CExplosion::GetExplosionType(uint8 id)
...
 {
 	return &gaExplosion[id].m_vecPosition;
 }
 
 bool
-CExplosion::AddExplosion(CEntity *explodingEntity, CEntity *culprit, eExplosionType type, const CVector &pos, uint32 lifetime, bool unk)
+CExplosion::AddExplosion(CEntity *explodingEntity, CEntity *culprit, eExplosionType type, const CVector &pos, uint32 lifetime, bool isMakeSound)
 {
+	const RwRGBA color = { 160, 160, 160, 255 };
+	const RwRGBA colorGrenade = { 96, 96, 96, 255 };
```
[the remark this answers is not in the document]
Comment (theR4K) [reply src/weapons/Explosion.cpp:105]: hmm, you right

File: src/audio/AudioLogic.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (erorcun) [reply src/audio/AudioLogic.cpp:5935]: I know

File: src/weapons/Explosion.cpp [hunk 4]
```diff
@@ -129,6 +140,7 @@ CExplosion::AddExplosion(CEntity *explodingEntity, CEntity *culprit, eExplosionT
 	explosion.m_nIteration = 1;
 	explosion.m_nActiveCounter = 1;
 	explosion.m_bIsBoat = false;
+	explosion.m_bIsMakeSound = makeSound;
```
Comment (erorcun) [inline src/weapons/Explosion.cpp:143]: please also change m_bIsMakeSound

Comment (theR4K) [reply src/weapons/Explosion.cpp]: I not think so

**Review by erorcun — APPROVED** [review]

**Review by ShFil119 — APPROVED** [review]
looks ok

**Pull request merged**